docs/debugger/dap.md

```diff
@@ -0,0 +1,35 @@
+# Debug Adapter Protocol
+
+[Debug Adapter Protocol](https://www.graalvm.org/latest/tools/dap/) is yet
+another instrument available for a Truffle language. The DAP is a native
+protocol for VSCode and as such, works only via VSCode. To start Enso with DAP
+server waiting for a client to attach, launch enso via:
+
+```
+env JAVA_OPTS='-Dpolyglot.dap' ./built-distribution/enso-engine-*/enso-*/bin/enso --run *.enso
+```
+
+Once DAP server is started and ready for a client to be attached, the following
+output will be printed:
+
+```
+[Graal DAP] Starting server and listening on /127.0.0.1:4711
+```
+
+In VSCode, create a new
+[Launch configuration](https://code.visualstudio.com/docs/editor/debugging#_launch-configurations)
```

Comment:
The configuration is already present in [VSCode configuration of Enso repository](https://github.com/enso-org/enso/blob/a123cd0d9f4b04d05aae7a5231efba554062188f/.vscode/launch.json#L13).

Please adjust the documentation to that. E.g. 
- point where the configuration is
- instruct people to open the workspace

Reply:
I didn't know we had this launch config in our repo. I have updated the docs in b3492b141362821d5b4526af72cb63454ec4753c.